assists/openamp-xlnx.py

```diff
@@ -24,6 +24,52 @@
 from lopper import Lopper
 from lopper import LopperFmt
 import lopper
+from lopper_tree import *
+
+openamp_file_name = "openamp_lopper_gen.h"
```

Comment:
Ben, Do we think that the filename might vary ? In that case, this can be the default and we could check the options dictionary in the assist callback for a filename. That way it could passed via a lop file and changed if needed.

Reply:
good point. added option to set file name via input r5 lop file in later commit